create_live_iso

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+bas64_str=`base64 -w 0 handle-user-data`
```

Comment:
Can we make this more generic? This will only work for handle-user-data. I think we will have more scripts in the end. I even would like to split the script handling the userdata from the one doing the formatting.

In this case one would first have to look at the (template) kickstart file and determine which scripts are included where. Then for each detected script we call the base64 encoding and insert the content.